docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -865,6 +865,9 @@ Given a distance symbol "dcode" (>= 16 + NDIRECT), and extra bits
    distance = ((offset + dextra) << NPOSTFIX) + lcode + NDIRECT + 1
 .fi
 
+If a decoded distance has a zero or negative value, the stream should
```

Comment:
Since only the first 16 distance symbols can resolve to zero or negative values, I think this paragraph should be moved up a bit, after line 843, something like:

"If a special distance symbol resolves to a zero or negative value, the stream should be rejected as invalid."